### ISSUE TITLE
REPL: Warning about underscore

### ISSUE THREAD

**Issue by xixixao** [jashkenas/coffeescript#4528]
I can't find an existing issue for this:
```
$ node -v
v7.9.0
$ coffee -v
CoffeeScript version 1.12.5
$ coffee
coffee> 4
Expression assignment to _ now disabled.
4
coffee>
```
Node REPL is constantly warning that assignment to underscore is now disabled. Latest Node, latest CS.

**Comment by vendethiel** [comment]
See https://github.com/nodejs/node/issues/5431 for rationale

**Comment by xixixao** [comment]
@vendethiel Might explaining specifically for CoffeeScript? I am not using any explicit _ assignment I'm aware of.

**Comment by vendethiel** [comment]
Look at the repl code. We generate an assignment to `_` to force it to be an expression.

**Comment by lydell** [comment]
Couldn't we assign to some other name, then?

**Comment by vendethiel** [comment]
Yes, we need to do this. My comment didn't mean we should close the issue.

**Comment by GeoffreyBooth** [comment]
We do this because the Node REPL itself treats `_` as a special case, assigning it to the last returned value:

```bash
$ node
> a = 1
1
> _
1
```

As @vendethiel noted above, see https://github.com/nodejs/node/issues/5431.

**Comment by xixixao** [comment]
Can we just assign to `__` ? This is super annoying in the REPL.

**Comment by GeoffreyBooth** [comment]
Resolved by #4559.
